server/rpc.go

```diff
@@ -35,6 +36,8 @@ const GRPCTimeout = 8 * time.Second
 
 var authTokenValidPeriod = 30 * time.Minute
 
+var discoveryAuthWebhookCache = cache.New(authTokenValidPeriod, authTokenValidPeriod)
```

Comment:
We might want the cleanup interval to be shorter otherwise in the worst case we could end up with cache items that expire after 30 minutes, but that are not cleaned up for awhile (i.e. something like 29 minutes).

Reply:
fixed in 7dd34f5